### ISSUE TITLE
Do we want to categorize Blog posts?

### ISSUE THREAD

**Issue by vegantriathlete** [vegantriathlete/how-to-brand-your-website#7]
It's easy to "tag" posts in Drupal. But, I'm wondering if it's useful for people to navigate to posts by category. I suppose it is, since that may be something that keeps people on the site. When they finish one blog post that they liked they could read others in that category.

However, it's also possible that I can have specific "You might also like . . ." type of recommendations for each blog post.

The idea of having the posts categorized does present another way that people can immerse themselves in the site. But, we'd encounter the issue of where we would place that tag cloud or category block. I really want the site to be optimized for mobile devices and I think that having a multi-column layout is going to be a problem. I guess we could have some type of "slide menu" that they could expose to get to stuff like that.

**Comment by kpalana** [comment]
This is an issue I also have had when redoing my website. I used to have categories and a tag cloud but felt they looked clunky on the new site. I left them out and opted to have "related posts" at the bottom of each individual blog post instead.

I recommend we try to address these things in as universal a way as possible. Meaning, as you outlined above, we address issues regardless of what tool or platform someone is using to build their website.

**Comment by vegantriathlete** [comment]
The downside of the related blog posts approach, is that I have to manually maintain that connection. It's not a hard thing to do. It's hard to remember all the posts after a while and be able to remember which of the older ones could be related to the newest one I'm adding.

I could still use categorization (tagging) and just programatically choose up to three random posts that use the same tag. Just because I tag the posts doesn't mean I have to show a tag cloud. Also, the tags themselves can be links. A reader could click one of the tags to get a set of paged results that use that tag.

So, I am thinking that I do want to go the route of "tagging" posts. I don't think it's a problem that I implement it using Drupal's tagging (specifically Taxonomy) system. If somebody is using a different platform that doesn't allow that type of tagging (although I can't imagine one that doesn't), they can still choose to implement the idea of "related posts" however they want. I don't want to limit the way I choose to build my website based on what somebody else using another platform can or cannot do. This is a real site that I'm building for my own real goals. Our focus is on branding, not on teaching people generic website building skills.

**Comment by kpalana** [comment]
True. I think some sort of tagging makes sense even if you aren't actively using it.
